build.gradle

```diff
@@ -22,8 +22,8 @@ dependencies {
     implementation 'com.github.jiconfont:jiconfont-google_material_design_icons:2.2.0.2'
 
     //Slf4j configuration
-    compile 'ch.qos.logback:logback-classic:1.2.3'
-    compile 'ch.qos.logback:logback-core:1.2.3'
+    testCompile 'ch.qos.logback:logback-classic:1.2.3'
```

Comment:
@mikera Sorry but I can not accept this commit because I'm woked on a Material logger to increase the debugging.

You can find the logger implementation here  https://github.com/vincenzopalazzo/material-ui-swing/blob/development/src/main/java/mdlaf/utils/MaterialLogger.java

Reply:
I don't think libraries should normally export logging implementations in their dependencies. This can cause issues for users that have different logging setups.

Shouldn't all debugging be done in `test` packages anyway? Having logging libraries as a test-only dependency seems sensible.

Reply:
@mikera yes before the last version of the library the logger was only inside the test package but now I need a logger because the library creates a memory leak error an I don't want use the System.out.println to debugging.

I agree with you, but at this moment is necessary to have a logger inside the library, after a developing case we can remove the class.

I hope you can understand the necessity

Reply:
Have the logging API inside the library, but keep the logback implementation in `test` scope is my recommendation.